ls8/server/app.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const { join } = require("path");
+// const cookieSession = require("cookie-session");
+const session = require("express-session");
+const cors = require("cors");
+const MongoStore = require("connect-mongo")(session);
+const api = require("./api/index");
+const { connectToDb } = require("./dbConnection");
+const { env, argv, platform } = process;
+const { log_module } = require("./api/log_module");
+
+require('dotenv').config()
+
+console.log("NODE_ENV:", !!Number(env.NODE_ENV));
+
+const server = express();
+connectToDb();
+
+server.use(cors({
+  origin: 'http://localhost:2530',
+  credentials: true
+}));
+
+
+server.use(
+  // cookieSession({
+  //   name: "session",
+  //   // Тоже так никогда не делайте!
+  //   keys: ["secret string"],
+  //   maxAge: 24 * 60 * 60 * 1000, // 24 hours
+  // })
+  session({
+    secret: "secret string",
```

Comment:
```suggestion
    secret: process.env.SECRET || "secret string",
```
На проде передавать через переменные окружения.